.github/CODEOWNERS

```diff
@@ -107,6 +107,9 @@
 /core/domain/voiceover_services*.py @sagangwee
 /core/templates/domain/opportunity/ @sagangwee
 /core/templates/pages/contributor-dashboard-page/ @sagangwee
+/core/templates/pages/contributor-dashboard-page/services/translate-text-backend-api.service.ts @sagangwee
+/core/templates/components/angular-html-bind/angular-html-bind-wrapper.directive.ts @sagangwee
+/core/templates/components/angular-html-bind/angular-html-bind-wrapper.directive.spec.ts @sagangwee
```

Comment:
Remove these two lines, they are already assigned to me down below

Reply:
Done